Sources/Filters+Strings.swift

```diff
@@ -120,13 +120,23 @@ extension Filters {
     }
 
     static func removeNewlines(_ value: Any?, arguments: [Any?]) throws -> Any? {
-      let removeSpaces = try Filters.parseBool(from: arguments, index: 0, required: false) ?? true
+      let mode = arguments.first as? String ?? "all"
```

Comment:
Shouldn't we throw if the argument is not a valid string, like `removeNewlines:42`?

So if there's no argument at all (`arguments.first == nil`), agreed that `all` should be the default. But if the argument isn't castable as a `as? String`, maybe throw?